dotnet-logging/StructuredLoggingUsingLog4Net/StructuredLoggingUsingLog4Net/Controllers/LogTestController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using log4net;
+using Microsoft.AspNetCore.Mvc;
+
+namespace StructuredLoggingUsingLog4Net.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class LogTestController : ControllerBase
+    {
+        private readonly ILog _logger;
+        public LogTestController(ILog logger)
+        {
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        }
+
+        [HttpGet]
+        public IActionResult Get()
+        {
+            _logger.Info("This is an informational log.");
+            _logger.Error("This is an error log.");
+            _logger.Info(new { id = 1, name = "John" });
+            return Ok("Test success!");
```

Comment:
We should have a newline between the return statement and the logs. All the log statements can be kept together.